src/Root.tsx

```diff
@@ -220,7 +223,9 @@ export default () => {
       };
 
       if (onboardingCompleted) {
-        if (status === 'active' && !appIsLoading) {
+        if (status === 'active' && checkingBiometricForSending) {
```

Comment:
When the touch ID is asked the app did not go into background for Android, this only happens on iOS for some reason.

`but the first verification will actually send the tx instead of unlocking the app` This is not happening to me, when going background the touch ID request is cancelled, the unlock via touch is requested and I have to slide to send again to send funds.

Reply:
Same issue with Android (S9+)

Reply:
Yes sorry, I could reproduce it now. Not sure how to solve it

Reply:
Solved